msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java

```diff
@@ -108,4 +113,9 @@ public IAccount getAccount() {
     public String[] getScope() {
         return mAccessToken.getTarget().split("\\s");
     }
+
+    @Override
+    public boolean isServicedFromCache() {
```

Comment:
Not about `is` vs `was` here....

Grammatically, I think `was` works better

```java
if (result.wasServicedFromCache()) {
    // ...
}
```

There may be other method names we could choose from....

Reply:
Done here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1048/commits/70be3c0419d2ad18ad565fdea69411540c038908

We have the same thing on the internal `ILocalAuthenticationResult` here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/internal/result/ILocalAuthenticationResult.java#L129 I'll change that too, albeit in a separate PR in common